qiita_pet/handlers/api_proxy/artifact.py

```diff
@@ -54,6 +54,9 @@ def artifact_summary_get_request(user_id, artifact_id):
     summary = artifact.html_summary_fp
     job_info = None
     errored_jobs = []
+    processing_jobs = [
+        [j.id, j.command.name, j.status, j.step] for j in artifact.jobs()
```

Comment:
What do you mean that the braces are off? It's a list comprehension generating a list of 4 elements, but filtering for those jobs that are part of an `artifact transformation` software. See [this line](https://github.com/biocore/qiita/pull/1706/files/22c5743ed31c12f7940eeb41a5c707136a3e8894#diff-4220e7483ec380c13e6819b835190569R136) for an example of the data generated here.